src/engraving/dom/dynamic.cpp

```diff
@@ -643,6 +648,8 @@ PropertyValue Dynamic::propertyDefault(Pid id) const
         return DynamicSpeed::NORMAL;
     case Pid::PLAY:
         return true;
+    case Pid::APPLY_TO_ALL_VOICES:
+        return false;
```

Comment:
Shouldn't the default rather be `true`, just like it was before your PR?

Reply:
It's false by design. If you don't like this new behavior, you can discuss it with @avvvvve in this issue
https://github.com/musescore/MuseScore/issues/19137